File: pset6/mario/less/mario.py
from cs50 import get_int
while True:
    h = get_int("Height: ")
    if h < 9 and h > 0:
        break
n = 1
m = 1
for i in range(h):
    for k in range(h-1):
        print(" ", end="")
    for j in range(n):
        print("#", end="")
    print("  ", end="")
    for l in range(m):
        print("#", end="")
    print()
    h -= 1
    n += 1
    m += 1

File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    int i, j, k, l;
    do//propmting user for positive integer
    {

        l = get_int("Height: ");

    }

    while (l < 1 || l > 8);

    for (i = 0; i < l; i++)//iteration statements

    {

        for (j = l; j > i + 1; j--)//iteration statements
        {

            printf(" ");

        }
        for (k = l - i; k <= l; k++)//iteration statements
        {
            printf("#");
        }
        printf("  ");
        for (k = l - i; k <= l; k++)//iteration statements
        {
            printf("#");
        }
        printf("\n");


    }

}